can/logger.py

```diff
@@ -201,12 +219,42 @@ def main() -> None:
     print(f"Connected to {bus.__class__.__name__}: {bus.channel_info}")
     print(f"Can Logger (Started on {datetime.now()})")
 
+    # argparser.py in Python 3.7.11 does not properly consider boolean
```

Comment:
You don't need any of this. Look at how the `--fd` argument is handled. It is also a boolean.

Reply:
I agree. I have modified my changes. This simplifies things. Thank you.